src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -156,7 +157,8 @@ void fillInnerParts() {
 
         // Must add a getFilteredProcedureList in logic for a single client-- works with the list method
         List<Procedure> procList = new ArrayList<>();
```

Comment:
Delete this example part before merging! You might actually have a merge conflict after I merge my listProc.